unlock-app/src/components/interface/checkout/main/Select.tsx

```diff
@@ -393,6 +399,37 @@ export function Select({ checkoutService }: Props) {
     })
   }
 
+  if (isLoadingWaas) {
```

Comment:
This should not be in the `Select` view IMO but one level "above".
The logic in this file here should be as much as possible only about selecting the lock.